<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preRun</class>
 <widget class="QWidget" name="preRun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры запуска</string>
  </property>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>118</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Продолжить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
